src/zoid/buttons/util.js

```diff
@@ -89,19 +89,23 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
             };
 
             session.onpaymentmethodselected = (e) => {
```

Comment:
Have you tried destructuring like this? Seems strange that the previous solution didn't work. Wondering if we need a fasly check on the args of these callbacks.
```js
session.onpaymentmethodselected = ({ paymentMethod }) => {
  listeners.paymentmethodselected({ paymentMethod });
};

Reply:
That works!  Thanks @gregjopa